deployment.rb

```diff
@@ -118,6 +118,7 @@ def load_configuration
     'pusher_app_id'               => 'fake_app_id',
     'pusher_application_key'      => 'fake_application_key',
     'pusher_application_secret'   => 'fake_application_secret',
+    'stub_school_data'            => [:adhoc, :test].include?(rack_env),
```

Comment:
OK, given that we will no longer be re-seeding after the first time that makes sense. Seed already takes so long on dev (and test locally) that I avoid it unless absolutely necessary.